src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -104,7 +104,8 @@ export class WdsDataTableProvider implements DataTableProvider {
 
   tsvFeatures: TSVFeatures = {
     needsTypeInput: true,
-    sampleTSVLink: 'https://storage.googleapis.com/terra-featured-workspaces/Table_templates/template_sample-wds-table.tsv', //TODO: This location may need to change
+    sampleTSVLink: 'https://azurefeaturedworkspace.blob.core.windows.net/featuredworkspacedata/template_data_table_Azure.txt',
+    dataImportSupportLink: '',
```

Comment:
if/when we have a WDS-specific support article, we can add its link here and then the UI will start showing the link again.